assets/src/components/d3/createResourceAccessChart.js

```diff
@@ -142,7 +142,7 @@ function createResourceAccessChart ({ data, width, height, domElement }) {
       .text(d => (
         ((mainYScale(d.resource_name) + mainYScale.bandwidth() / 2) < miniHeight) &&
         ((mainYScale(d.resource_name) + mainYScale.bandwidth() / 2) > 0))
-        ? d.total_count
+        ? d.total_percent
```

Comment:
Were we going to add the percent sign here? (e.g. `d.total_percent  + '%'`)

Reply:
Yeah I had a new issue #887 filed about this one

Reply:
Seems related enough to your other changes that it would fit here.

Reply:
I just thought it might have been controversial so didn't want to sneak it in here. ;)

Reply:
That's fine I can add it